scripts/installscripts/install-jukebox.sh

```diff
@@ -1082,40 +1089,25 @@ install_main() {
     sudo chown mpd:audio "${mpd_conf}"
     sudo chmod 640 "${mpd_conf}"
 
-    # start mpd
-    echo "Starting mpd service..."
-    sudo service mpd restart
-    sudo systemctl enable mpd
 
     # Spotify config
     if [ "${SPOTinstall}" == "YES" ]; then
         echo "Configuring Spotify support..."
-        local etc_mopidy_conf="/etc/mopidy/mopidy.conf"
-        local mopidy_conf="${HOME_DIR}/.config/mopidy/mopidy.conf"
+        local mopidy_conf="/etc/mopidy/mopidy.conf"
         sudo systemctl disable mpd
-        sudo service mpd stop
+        sudo systemctl stop mpd
         sudo systemctl enable mopidy
+        sudo systemctl stop mopidy
         # Install Config Files
         sudo cp "${jukebox_dir}"/misc/sampleconfigs/locale.gen.sample /etc/locale.gen
         sudo cp "${jukebox_dir}"/misc/sampleconfigs/locale.sample /etc/default/locale
         sudo locale-gen
-        mkdir -p "${HOME_DIR}"/.config/mopidy
-        sudo cp "${jukebox_dir}"/misc/sampleconfigs/mopidy-etc.sample "${etc_mopidy_conf}"
-        cp "${jukebox_dir}"/misc/sampleconfigs/mopidy.sample "${mopidy_conf}"
+        sudo cp "${jukebox_dir}"/misc/sampleconfigs/mopidy.conf.sample "${mopidy_conf}"
         # Change vars to match install config
-        sudo sed -i 's/%spotify_username%/'"$SPOTIuser"'/' "${etc_mopidy_conf}"
```

Comment:
Did you also fix the config issue? AFAIR the two mopidy.conf files were needed, because one was used by mopidy and the other one by Iris?

Reply:
Good hint. i will check that again.

Reply:
To be honest um not really sure how it works, but we have a lot of issues where users had problems and, if I remember correctly the two configs were actually needed.

If we could fix that it would be helpful.

Reply:
This looks all OK to me. I can't see why you needed the other config file in $HOME unless you were also running Mopidy as a user (e.g. not a service) somewhere? I can't see where that would be but I am not very familiar with how this fits together.

Reply:
So keep the changes ;)